src/drivers/tap_esc/tap_esc.cpp

```diff
@@ -869,7 +869,7 @@ TAP_ESC::ioctl(file *filp, int cmd, unsigned long arg)
 
 	case MIXERIOCLOADBUF: {
 			const char *buf = (const char *)arg;
-			unsigned buflen = strnlen(buf, 1024);
+			unsigned buflen = strlen(buf);
```

Comment:
I do see there is a strnlen version in qurt, any reason why it is needed for qurt?

Reply:
@Lampoo Can you point me to that source?

Reply:
@ChristophTobler I can see it is at :src/platforms/qurt/px4_layer/px4_qurt_impl.cpp:148:size_t strnlen(const char *s, size_t maxlen)